.storybook/themes/commerce.scss

```diff
@@ -288,6 +289,8 @@
 					),
 				),
 				quilted-image: (
+					margin: 0 auto,
+					max-width: var(--layout-content-max-width),
```

Comment:
```suggestion
					max-width: var(--layout-content-max-width + 0px),
```

Reply:
This will allow you to use a unitless `--layout-content-max-width`

Reply:
@nschubach : Is that working for you? I've tried a unitless `layout-content-max-width` and used...
```
quilted-image: (
               margin: 0 auto,
               max-width: var(--layout-content-max-width + 0px),
```
... and it's still resulting in the 1440px width.
Also tried `calc(var(--layout-content-max-width) + 0px)` and having the same result.

Reply:
@nschubach : Actually, multiplication worked for me.

Reply:
Sorry, I was somewhat guessing and didn't realize I missed the `var()` or that + would not work in that case.  I was going ot say that you could use the `--layout-content-scale-width` but that would only apply for the desktop.  The mobile is full width from the looks of figma.  Though, I'm curious how this lines up with that given the different calculation.  We'll have to put them on a page together.

Reply:
multiplication's working for me as well. want me to push it up @ecarlisle ?